applications/butler/values.yaml

```diff
@@ -67,3 +67,7 @@ config:
   # -- URI to the file specifying the DirectButler configuration to be used
   # by the butler server
   configUri: ""
+  # -- The prefix of the path portion of the URL where the Butler service will
+  # be exposed.  For example, if the service should be exposed at
+  # `https://data.lsst.cloud/api/butler`, this should be set to `/api/butler`
+  endpointPathPrefix: "/api/butler"
```

Comment:
We've used `pathPrefix` fairly uniformly elsewhere, so I would recommend that as well here.